content/user-management copy.md

```diff
@@ -0,0 +1,57 @@
+   - IdentifiedUsers: Identified users are what the name implies: users who we know. These can be intranet users synchronized from the Active Directory or users created for your clients or partners. The membership of this group depends totally on you, the system does not add anybody to this group automatically.
+   - RegisteredUsers: The Registered users group is a regular group, you may modify its members freely; the only specialty of it is that the sample public registration workflow adds users to it. They should have slightly less permissions than identified users (see above) because the registration can be very open in a public portal. This is why there are not many permission settings for this group in the default structure.
+   - Somebody: This is a low-level placeholder user for cases when we cannot find a creator/modifier user that previously existed or the currently logged in user does not have permissions to see the creator or modifier of a content. Please do not delete this user and do not try to use it as a regular user.
+
+# Built-in public groups/roles for basic tasks
+
+Administrators
+
+Administrators considered to be the most powerful users in the system. This may be true for some features but not necessarily for all content. In the default structure, administrators have access to everything, but when you build your project and add your custom content, it is perfectly fine if you hide something from the Administrators group. For example you may remove their permissions from confidential documents completely.
+
+By default only the Admin user is a member of this group, but you can add more members as you like. Please use this group for administrative permissions instead of single users.
+
+Editors
+
+We do not have any predefined permissions for this group by default, we consider them as a middle layer between administrators and regular users. For example you can define permissions for the global Editors group for skins and renderers or content types.
+
+
+- Developers: We created this group to help developers work in the development phase of a new project. Group members will see the PRC on the pages, will have access to the Content Explorer. By the default installation the group is the member of the Administrators group.
+
+# Permission
+sensenet offers you a sophisticated permission system, giving you the possibility to apply permission rules according to the logic you prefer.
+Using role-based permissions, you can assign users to roles taking advantage of predefined permission settings. Role based access control provides management of users and tasks that they can perform on contents (for example built-in permissions such as See, Open, Save, Delete ect, and it is also possible to add custom ones as well.
```

Comment:
_Role based access control provides management of users and tasks that they can perform on contents_

this is not clear for me, we should discuss about it